index.js

```diff
@@ -44,6 +44,7 @@ module.exports = function (config) {
       .option('-o, --open', 'Open the Web GUI after startup')
       .option('-v, --verbose')
       .option('--silent')
+      .option('-j, --disable-javascript', 'If set, disables the execution of JavaScript in Mails')
```

Comment:
@oktapodia @donut87, I love this change, but it's got me thinking - should JavaScript be disabled by default as a typical email client would strip out all JS code automatically? This would make this flag something like `-js, --enable-javascript, If set, enables JavaScript execution in emails`